packages/apps/plugins/plugin-markdown/src/components/Layout.tsx

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React, { type PropsWithChildren } from 'react';
+
+import { Main } from '@dxos/react-ui';
+import { baseSurface, mx, textBlockWidth, topbarBlockPaddingStart } from '@dxos/react-ui-theme';
+
+export const MainLayout = ({ children }: PropsWithChildren) => {
```

Comment:
note to think about factoring out common containers across plugins for uniformity.

Reply:
I've wondered about this. @thure do you think that the layout plugin should just include these styles around the main surface?

Reply:
I’m not entirely sure I’m on the same page as either of you… I think we have a meaningful separation of concerns between `Main.Content` and however an app plugin might want to lay out content in there.

1) `flex flex-col` can almost always be removed (block layout is usually fine and introduces less risk of introducing subtle bugs)
  - in this case it’s used so the viewport height is filled, but that is redundant with the `bounce` flag on `Main.Content`
2) `textBlockWidth` is a meaningful extrinsic size that doesn’t need to be a component because it is more useful as a fragment
3) Everything else here is spacing, which is highly sensitive to context.
4) AFACIT, at least one of these `<div>`s can be removed.

Reply:
`Main.Content` probably doesn’t need `baseSurface` either, since it is never superimposed onto anything.

Reply:
> Everything else here is spacing, which is highly sensitive to context

I guess this is why I was wondering if the layout should wrap the main surface w/ a container that can help. Otherwise the plugins need to know about the height of the navbar, whether the sidebar is open or not, how big is the sidebar, etc.